core/assets/bundles/bundle_fr.properties

```
@@ -149,7 +149,7 @@ invalidid = ID du client invalide! Veuillez soumettre un rapport d'erreur.
 server.bans = Bannis
 server.bans.none = Aucun joueur banni trouvé!
 server.admins = Administrateurs
-server.admins.none = Pas d'administrateurs trouvés!
+server.admins.none = Pas d'administrateur trouvé!
```

Cette tournure de phrase me semble plus correcte, et est plus consistente avec la traduction de la ligne 150.
```suggestion
server.admins.none = Aucun administrateur trouvé!
```

---

Je suis d'accord